### PR TITLE
Add cardscan react native

### PR DESCRIPTION
**Pull request by awushensky**
For development environment setup, I followed this guide using React Native CLI Quickstart:
https://reactnative.dev/docs/environment-setup

using the `minimal` template

for integration of cardscan library, I followed this guide:
https://github.com/getbouncer/react-native-cardscan

Project was set up using the following command:
```
npx react-native init cardscanreactnativetest2 --version 0.61.5
```

Then added the cardscan module with:
```
npm install getbouncer/react-native-cardscan
```

Launched cardscan app with:
```
npx react-native run-android
adb reverse tcp:8081 tcp:8081 && adb shell am start -n com.cardscanreactnativetest2/com.cardscanreactnativetest2.MainActivity
```

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -21,7 +21,6 @@ buildscript {
 
 allprojects {
     repositories {
-        mavenLocal()
```
Comment (awushensky) [inline android/build.gradle:24]: removed this because it was picking up incompatible dev builds from local maven. This probably can stay otherwise.

File: android/app/build.gradle [hunk 2]
```diff
@@ -181,6 +181,11 @@ android {
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation "com.facebook.react:react-native:+"  // From node_modules
+    implementation "androidx.swiperefreshlayout:swiperefreshlayout:1.0.0"
```
Comment (awushensky) [inline android/app/build.gradle:184]: this is unrelated to cardscan, and was added to make react-native happy.

File: android/app/src/main/java/com/cardscanreactnativetest2/MainApplication.java [hunk 3]
```diff
@@ -44,6 +46,7 @@ public void onCreate() {
     super.onCreate();
     SoLoader.init(this, /* native exopackage */ false);
     initializeFlipper(this); // Remove this line if you don't want Flipper enabled
+    ScanActivity.apiKey = "qOJ_fF-WLDMbG05iBq5wvwiTNTmM2qIn";
```
Comment (awushensky) [inline android/app/src/main/java/com/cardscanreactnativetest2/MainApplication.java:49]: Make sure to replace this with your own apiKey. this one will only work for a short time.

File: android/app/build.gradle [hunk 4]
```diff
@@ -181,6 +181,11 @@ android {
...
+
+    implementation 'com.getbouncer:cardscan-base:1.0.5144'
+    implementation 'com.getbouncer:cardscan:1.0.5144'
```
Comment (San10694) [inline android/app/build.gradle:187]: why cardscan-base has been added here ,if its already there in the node_modules/react-native-cardscanner/android/build.gradle
